### ISSUE TITLE
Foxaholic Cloudflare Infinite Loop

### ISSUE THREAD

**Issue by nashahzad** [LNReader/lnreader-plugins#828]
### Source information

Foxaholic

### Steps to reproduce

1. Go to Foxaholic source
2. Open the webview for foxaholic
3. Click checkbox to verify as human for cloudlfare
4. Be stuck in an infinite loop on step 3

### Expected behavior

In the past after step 3 it would normally get past cloudlfare and load foxaholic.

### Actual behavior

Never gets past step 3 of clicking the checkbox to verify as human for cloudflare.

### LNReader version

1.1.19

### Android version

Android 14

### Other details

I've done a fresh re-install of LNReader to see if that would help and it did not.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[1.1.12](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/LNReader/lnreader/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by error7404** [comment]
it's fixed in beta

**Comment by nashahzad** [comment]
> it's fixed in beta

ohh, thank you! I didn't check out the beta version of the app. I've only been using whatever I see as the latest release.

**Comment by error7404** [comment]
https://github.com/LNReader/lnreader/releases/tag/v2.0.0-beta.0
make sure to convert your backup

**Comment by nyagami** [comment]
You should use v2 beta app or wait for its stable release
